product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/06_pgbouncer_configuration.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Configuring EDB PgBouncer
+---
+
+[EDB PgBouncer](/pgbouncer/latest/) is a lightweight connection pooling utility that you can connect to your BigAnimal cluster. It saves you time by pooling previously established connections to your server. Instead of waiting for your server to negotiate every connection, EDB PgBouncer can negotiate the connection for you. 
```

Comment:
We need to be careful to not overload the term "connect" and "connection" in this topic. 

```suggestion
[EDB PgBouncer](/pgbouncer/latest/) is a lightweight connection pooling utility that you can use to more quickly connect to your BigAnimal cluster. It saves you time by pooling previously established connections to your server. Instead of waiting for your server to negotiate every connection, EDB PgBouncer can negotiate the connection for you. 
```
Again, I am not sure this is technically correct.